assets/js/marker.js

```diff
@@ -407,8 +407,15 @@ class Marker {
       .end()
       .find('img.background').attr('src', bgUrl);
   }
-  updateOpacity(opacity = Settings.markerOpacity) {
-    this.lMarker && this.lMarker.setOpacity(this.canCollect ? opacity : opacity / 3);
+  updateOpacity(opacity = Settings.markerOpacity, isInvisibleRemovedMarkers = Settings.isInvisibleRemovedMarkers) {
+    let targetOpacity;
+    if (this.canCollect) {
+      targetOpacity = opacity;
+    } else {
+      targetOpacity = (isInvisibleRemovedMarkers? 0: opacity / 3);

```

Comment:
Brackets are unnecessary here, add also space before `?` and `:` in ternary operator.

Reply:
> I will merge your PR, but i don't really like the idea make markers only transparent, marker should be removed from the map, because you can still click transparent markers. In the future I may replace your idea with permanent removing markers from the map.

Totally agreed. I also thought it would be much better if the markers got "removed" not hidden, though, I was just being conservative.

> In the future I may replace your idea with permanent removing markers from the map.
I think I'll give that a try myself.